tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker-test.php

```
@@ -512,6 +536,314 @@ function ( $pre, $args, $url ) {
+	 * @return void
+	 */
+	public function test_generate_archive_feed_returns_finalized_feed() {
+		$product = $this->create_simple_product_with_stock( 5 );
+		$this->sut->track_product_archive( $product->get_id() );
+
+		$feed = $this->sut->generate_archive_feed();
+
+		$this->assertInstanceOf( WC_Stripe_Agentic_Commerce_Csv_Feed::class, $feed );
+		$this->assertNotNull( $feed->get_file_path() );
+		$this->assertFileExists( $feed->get_file_path() );
+	}
```

PHPMD (2.15.0)

[error] The method test_generate_archive_feed_returns_finalized_feed is not named in camelCase. (undefined)

(CamelCaseMethodName)